pkg/devices/devices.css

```diff
@@ -0,0 +1,68 @@
+@import "/page.css";
+@import "/listing.css";
+@import "/table.css";
+
+.app-devices {
+    display: block;
+}
+
+.content-header-extra {
```

Comment:
This is old style CSS. Could you use the ```.content-filter``` stuff you see in ```pkg/docker/docker.css``` instead? Sorry for the confusion. We should soon migrate that to a reusable component with reusable CSS, but for now lets just copy and paste it.